### PR TITLE
access token repository

### PR DESCRIPTION
**Pull request by Red-C**
create access token repository

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/Big-Lemon/ParkingFinder/pull/2" target="_blank">ReviewNinja

File: ParkingFinder/base/errors.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+class AuthorizationError(BaseError):
+    """
+    Invalid Access Token Exception
+
+    """
+    error = 'Invalid Access Token'
+
+
+class NotFound(BaseError):
```
Comment (4solo) [inline ParkingFinder/base/errors.py]: [style] rename as NotFoundError.

File: ParkingFinder/base/errors.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    """
+    Generic Not Found Exception
+    """
+    error = 'Not Found Exception'
+
+
+class InvalidArguments(BaseError):
```
Comment (4solo) [inline ParkingFinder/base/errors.py]: [style] rename as InvalidArgumentsError.

File: ParkingFinder/mappers/__init__.py [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+            raise NotImplementedError
+
+    @classmethod
+    def to_entity(cls, record):
+        params = cls.build_map(record)
+        return cls._ENTITY(params)
+
+    @staticmethod
+    @abstractmethod
+    def build_map(record):
```
Comment (4solo) [inline ParkingFinder/mappers/__init__.py]: private method

File: ParkingFinder/base/async_db.py [hunk 4]
```diff
@@ -0,0 +1,24 @@
+from contextlib import contextmanager
+from tornado.gen import Return
+
+from ParkingFinder.base import Session
```
Comment (Red-C) [inline ParkingFinder/base/async_db.py]: this one doesn't have to change. the import structure will be 
python internal packages
<-----white line
third party packages
<-----white line
packages from our project.

File: ParkingFinder/base/errors.py (hunk 1 above)
Comment (Red-C) [inline ParkingFinder/base/errors.py]: I change AuthorizationError to Unauthorized. all internal generic errors in our project are naming without 'Error' as ending in order to distinguish from system error.

File: ParkingFinder/base/errors.py (hunk 2 above)
Comment (Red-C) [inline ParkingFinder/base/errors.py]: save as above

**Pull request merged**